Integrations/QuinC/README.md

```diff
@@ -0,0 +1,276 @@
+## Overview
+---
+Use the QuinC integration to protect against and provide additional visibility into phishing and other malicious email attacks.
+This integration was integrated and tested with version 20190926 of QuinC
+## QuinC Playbook
+---
+For example, you can look at "Accessdata: Dump memory for malicious process" playbook to understand how to use this integration.
```

Comment:
Write here the name of the playbooks and their description. Users will look here and read which playbooks related to Accessdata and what those playbooks do.